internal/fetch/jobDescription_test.go

```diff
@@ -0,0 +1,45 @@
+package fetch
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func Test_parseJobDescription(t *testing.T) {
+	for _, tc := range []struct {
+		jobDescription    string
+		defaultPipelineID int64
+		want              *JobDescription
+		wantErr           bool
+	}{
+		{
+			jobDescription:    `sd@123:myName`,
+			defaultPipelineID: 999,
+			want: &JobDescription{
+				External:   `sd@123:myName`,
+				PipelineID: 123,
+				JobName:    "myName",
+			},
+		},
+		{
+			jobDescription:    `myName`,
+			defaultPipelineID: 123,
+			want: &JobDescription{
+				External:   `myName`,
```

Comment:
I feel like `MetaFileName` instead of `External` might make the code clearer, since that's what it really is.

Reply:
Done